lib/senkyoshi.rb

```diff
@@ -103,7 +111,7 @@ def self.initialize_course(canvas_file_path, blackboard_file_path)
     end
   end
 
-  def true?(obj)
+  def self.true?(obj)
```

Comment:
This change is unnecessary. In the files you are using this, add to the class you are in `include Senkyoshi` and you will have access to this helper. See `outcome_definition.rb:6`